scripts/requirements-run-test.txt

```diff
@@ -13,3 +13,6 @@ cbor>=1.0.0
 
 # use for twister
 psutil
+
+# pytest plugin
+-e ${ZEPHYR_BASE}/scripts/pylib/pytest-twister-ext
```

Comment:
pip install -r scripts/requirements-run-test.txt fails for me with this line

Reply:
`ZEPHYR_BASE=<path to Zephyr dir>` must be exported when installing `pytest-twister-ext` from requirements file. Or just install it manually with `pip install -e scripts/pylib/pytest-twister-ext`